### ISSUE TITLE
How to have a public calendar?

### ISSUE THREAD

**Issue by Benjamin-Loison** [Benjamin-Loison/android_packages_apps_Etar#14]
Related to [Benjamin_Loison/android/issues/4](https://codeberg.org/Benjamin_Loison/android/issues/4).

+47

**Comment by Benjamin-Loison** [comment]
For people to track my public reminders like [YouTube-operational-API/issues/301#issuecomment-2347783112](https://github.com/Benjamin-Loison/YouTube-operational-API/issues/301#issuecomment-2347783112).

**Comment by Benjamin-Loison** [comment]
Notably maybe my geographical reminders but have to pay attention to specify privacy friendly labels and maybe only let know that I have moved once this movement is finished or/and do not specify busy, like I am back home such that people possibly knowing my address don't know when to harm my home.

Tracked at [Benjamin_Loison/Reminders/issues/3](https://codeberg.org/Benjamin_Loison/Reminders/issues/3).

**Comment by Benjamin-Loison** [comment]
Maybe do not specify busy or out of home in case people know where I leave to harm my home.

**Comment by Benjamin-Loison** [comment]
Without having to use a dummy account.

**Comment by Benjamin-Loison** [comment]
Related to [Benjamin_Loison/framadate/issues/17](https://framagit.org/Benjamin_Loison/framadate/-/issues/17).

**Comment by Benjamin-Loison** [comment]
Related to [Benjamin_Loison/zm-build/issues/](https://codeberg.org/Benjamin_Loison/zm-build/issues){[6](https://codeberg.org/Benjamin_Loison/zm-build/issues/6),[4](https://codeberg.org/Benjamin_Loison/zm-build/issues/4)}.

![image](https://github.com/user-attachments/assets/dce4a589-4984-4be4-a4b5-41d3fd982341)

**Comment by Benjamin-Loison** [comment]
My work Zimbra instance is not appropriate because it is not for personal stuff I guess but also as would leak to some extent my location.

**Comment by Benjamin-Loison** [comment]
Except EteSync, Etar default calendar LineageOS app only supports CalDAV. Should investigate the latter to avoid relying even more on DAVx⁵.

I already read [Wikipedia: CalDAV (1240719232)](https://en.wikipedia.org/w/index.php?title=CalDAV&oldid=1240719232).

> It extends the [WebDAV](https://en.wikipedia.org/wiki/WebDAV) ([HTTP](https://en.wikipedia.org/wiki/HTTP)-based protocol for data manipulation) specification and uses the [iCalendar](https://en.wikipedia.org/wiki/ICalendar) format for the calendar data.

> For access control the concept of [ACLs](https://en.wikipedia.org/wiki/Access_control_list) are used, so each operation (view, edit, delete etc.) can be denied or granted per user. Therefore, the specification requires that CalDAV servers must support "WebDAV Access Control Protocol" ([RFC 3744](https://datatracker.ietf.org/doc/html/rfc3744)).

> The protocol also supports non-personal calendars, such as calendars for sites or organizations.

[Wikipedia: Comparison of CalDAV and CardDAV implementations#Server implementations (1263842289)](https://en.wikipedia.org/w/index.php?title=Comparison_of_CalDAV_and_CardDAV_implementations&oldid=1263842289#Server_implementations)
*RFC 4791 Cal­DAV*

**Comment by Benjamin-Loison** [comment]
Would be interesting to share a calendar per (group of) person(s) to provide a way for people to track events specific to me and the (group of) person(s) like giving back something after having found a workaround. To avoid linear number of calendars in the number of (group of) person(s) could encrypt an event and share the mean of decryption to the (group of) person(s). Such a linearity may be fine, as it is how messaging platforms work.

**Comment by Benjamin-Loison** [comment]
Related to #44.

**Comment by Benjamin-Loison** [comment]
<details>
<summary>Server implementations:</summary>
 
- [Wikipedia: DAViCal](https://en.wikipedia.org/wiki/DAViCal) ([davical](https://gitlab.com/davical-project/davical) has 100 GitLab stars and last commit was 1 month ago)
- [Wikipedia: Group-Office](https://en.wikipedia.org/wiki/Group-Office) ([groupoffice](https://github.com/intermesh/groupoffice) has 210 stars and last commit was 4 days ago)
- [Wikipedia: Horde (software)](https://en.wikipedia.org/wiki/Horde_(software)) ([horde/base](https://github.com/horde/base) has 50 stars and last commit was 3 years ago)
- [Wikipedia: Nextcloud](https://en.wikipedia.org/wiki/Nextcloud) ([nextcloud/server](https://github.com/nextcloud/server) has 28.8k stars and last commit was 8 minutes ago)
- [Wikipedia: OwnCloud](https://en.wikipedia.org/wiki/OwnCloud) ([owncloud/core](https://github.com/owncloud/core) has 8.5k stars and last commit was 4 days ago)
- [Wikipedia: SabreDAV](https://en.wikipedia.org/wiki/SabreDAV) ([sabre-io/dav](https://github.com/sabre-io/dav) has 1.6k stars and last commit was 3 months ago)
- [Wikipedia: Zimbra](https://en.wikipedia.org/wiki/Zimbra) ([zm-build](https://github.com/Zimbra/zm-build) has 195 stars and last commit was 2 months ago)
- [Wikipedia: Mac OS X Server#Address Book Server (1246582984)](https://en.wikipedia.org/w/index.php?title=Mac_OS_X_Server&oldid=1246582984#Address_Book_Server)
- [Wikipedia: Cyrus IMAP server](https://en.wikipedia.org/wiki/Cyrus_IMAP_server) ([cyrus-imapd](https://github.com/cyrusimap/cyrus-imapd) has 569 stars and last commit was 3 hours ago)
</details>

I otherwise thought about setting a `.ics` somewhere behind WebDAV but my calendar client might not support it.
The issue may be about giving read access to only this file and disable write access.
Well for a WebDAV instance dedicated for this file it may be easier, as do not need file precision for permissions.

**Comment by Benjamin-Loison** [comment]
I completely read [Wikipedia: Nextcloud (1274474064)](https://en.wikipedia.org/w/index.php?title=Nextcloud&oldid=1274474064) except *History*.

**Comment by Benjamin-Loison** [comment]
No field of the comparison table seems to state whether can make that an authenticated person can edit a calendar but otherwise it is read-only publicly. I know that Zimbra can do so, so maybe will try to stick to it even if it brings too many other features but as I would be the only one with an account it seems, it does not seem to be an issue.

**Comment by Benjamin-Loison** [comment]
Maybe the Zimbra instance I have in mind runs the closed-source edition.

*About* shows *Zimbra 10.0.13_GA_4736 (build 20250124040958)* that is 24/01/25 it seems.

**Comment by Benjamin-Loison** [comment]
> The Zimbra Web Client is a full-featured collaboration suite that supports email and group calendars.

Source: [Wikipedia: Zimbra (1254433276)](https://en.wikipedia.org/w/index.php?title=Zimbra&oldid=1254433276)

I completely read this article:

*Included open source projects* does not seem to mention anything about calendar.

> It supports CalDAV

> Zimbra can synchronize [...] calendar items with open-source mail clients such as [Mozilla Thunderbird](https://en.wikipedia.org/wiki/Mozilla_Thunderbird) and [Evolution](https://en.wikipedia.org/wiki/Evolution_(software)) and also with proprietary clients such as [Microsoft Outlook](https://en.wikipedia.org/wiki/Microsoft_Outlook) and [Apple Mail](https://en.wikipedia.org/wiki/Apple_Mail), either through proprietary connectors or using the [ActiveSync](https://en.wikipedia.org/wiki/ActiveSync) protocol,[[14]](https://en.wikipedia.org/wiki/Zimbra#cite_note-14) both available exclusively in the commercially supported version. Zimbra also provides native two-way sync to many mobile devices.[[9]](https://en.wikipedia.org/wiki/Zimbra#cite_note-VMware_Zimbra-9)

it a bit unclear but let us give it a try.

**Comment by Benjamin-Loison** [comment]
https://www.zimbra.com/product/download/

> A NEW V10.1 LICENSE KEY is required to run Zimbra Daffodil

https://www.zimbra.com/product/licenses-and-terms-of-use/

**Comment by Benjamin-Loison** [comment]
```bash
apt list | grep zimbra
```

does not return anything.

**Comment by Benjamin-Loison** [comment]
[Benjamin-Loison/android_packages_apps_Etar/issues/23](https://github.com/Benjamin-Loison/android_packages_apps_Etar/issues/23) would help.

**Comment by Benjamin-Loison** [comment]
To have a history, as I regularly postpone as I am running out of time, could leverage file system notification and commit each time, same for `crontab` updates and similar.

Related to [Benjamin_Loison/cron/issues/6](https://codeberg.org/Benjamin_Loison/cron/issues/6).

**Comment by Benjamin-Loison** [comment]
https://lemnoslife.com/Benjamin_Loison.ics

**Comment by Benjamin-Loison** [comment]
https://calendar.proton.me is probably encrypted:

*Settings* > *All settings* > *Proton Calendar* > *Calendars* > *Calendar*

> Share calendar

> Upgrade to a Mail paid plan to share your calendar.

**Comment by Benjamin-Loison** [comment]
Could try a free and open-source website providing the service I am looking for.

**Comment by Benjamin-Loison** [comment]
![Image](https://github.com/user-attachments/assets/5a9ebb16-91a0-4761-92c5-a5ac7362268e)

![Image](https://github.com/user-attachments/assets/92fcdde6-270f-4a40-afea-a1519d8b0a4a)

specifying arbitrary credentials leads to:

![Image](https://github.com/user-attachments/assets/d39fd2af-709e-4a41-94ba-813a96a396b8)

![Image](https://github.com/user-attachments/assets/dd16c7da-e000-46d1-910e-541c04800fce)

faces the same error.

**Comment by Benjamin-Loison** [comment]
Maybe EteSync calendar can be public even if it is somehow inappropriate.
I verified and we can only share it to an existing member, so it looks too complex.

**Comment by Benjamin-Loison** [comment]
DuckDuckGo and Google search *public free open-source calendar server* returns server implementations, not servers it seems.

**Comment by Benjamin-Loison** [comment]
DuckDuckGo search *Linux install Zimbra*.

**Comment by Benjamin-Loison** [comment]
https://inguide.in/install-zimbra-on-ubuntu-20-04-step-by-step/ seems to avoid compilation without using homemade scripts.

**Comment by Benjamin-Loison** [comment]
https://files.zimbra.com/downloads/ *Access Denied*
https://files.zimbra.com does not mention *downloads*.

**Comment by Benjamin-Loison** [comment]
https://www.zimbra.com/product/download/zimbra-collaboration-network-edition/ > *10.1.0 GA Release*

> Ensure that you have the new V10.1 license before you start the installation/upgrade process. Licenses for earlier editions will not work with V10.1.

https://www.zimbra.com/connect/forms/?form=trial-license

*What is your existing solution?* mentions *Zimbra Open-Source*.

**Comment by Benjamin-Loison** [comment]
https://files.zimbra.com/downloads/8.8.15_GA/ results in random file name being empty.

**Comment by Benjamin-Loison** [comment]
```bash
curl -I 'https://files.zimbra.com/downloads/8.8.15_GA/zcs-8.8.15_GA_4179.UBUNTU20_64.20211118033954.tgz'
```
<details>
<summary>Output:</summary>

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0  235M    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
HTTP/1.1 200 OK
Content-Type: application/x-tar
Content-Length: 246925287
Connection: keep-alive
Date: Sun, 09 Mar 2025 00:03:00 GMT
Last-Modified: Wed, 09 Aug 2023 07:17:04 GMT
ETag: "3a7eaf2a6ebcfd12e239ed0adb4d1977-30"
x-amz-server-side-encryption: AES256
Accept-Ranges: bytes
Server: AmazonS3
X-Cache: Hit from cloudfront
Via: 1.1 848dacc1a0789a0c23dff05db236ea62.cloudfront.net (CloudFront)
X-Amz-Cf-Pop: CDG50-C2
X-Amz-Cf-Id: _A9oknJTp_TgA5BSQhuj7ubP5wJGlfkYNJ63w74pPf9XxsJfpfhaaA==
Age: 6

```
</details>

**Comment by Benjamin-Loison** [comment]
Can't guess the up-to-date download URL it seems.

**Comment by Benjamin-Loison** [comment]
> Now, you need to download Zimbra OSE with the following command. You can always get the latest file from [Zimbra Website](https://www.zimbra.com/downloads/zimbra-collaboration-open-source/).

> Apologies, but the page you requested could not be found.

**Comment by Benjamin-Loison** [comment]
https://web.archive.org/web/20230801163144/https://www.zimbra.com/downloads/zimbra-collaboration-open-source/ seems to be last working snapshot.

**Comment by Benjamin-Loison** [comment]
It was still *8.8.15* but supported Ubuntu 20.04.

https://www.zimbra.com/downloads/zimbra-collaboration-open-source/archives/ does not work too.

**Comment by Benjamin-Loison** [comment]
DuckDuckGo search *zimbra open source*.

**Comment by Benjamin-Loison** [comment]
https://www.zimbra.com/product/enterprise-collaboration/#edition-comparison

> Zimbra caters to a diverse group of users with varying collaboration requirements. You can buy a combination of both Professional and Standard editions in a single license to meet those requirements.

**Comment by Benjamin-Loison** [comment]
*10.0.0* is only mentioning Ubuntu 20, so maybe let us look for another server implementation.

**Comment by Benjamin-Loison** [comment]
> license key send via email by zimbra team.

Source: https://vkttech.com/step-by-step-guide-to-installing-zimbra-10-on-ubuntu-20-04/

**Comment by Benjamin-Loison** [comment]
DuckDuckGo search *CalDAV Apache*.

**Comment by Benjamin-Loison** [comment]
https://owncloud.com
https://nextcloud.com

do not seem to propose such service.

**Comment by Benjamin-Loison** [comment]
https://framagenda.org may be what I am looking for. It is running on NextCloud.

**Comment by Benjamin-Loison** [comment]
https://framagenda.org/remote.php/dav/public-calendars/XXXXXXXXXXXXXXXX

> This is the WebDAV interface. It can only be accessed by WebDAV clients such as the Nextcloud desktop sync client.

**Comment by Benjamin-Loison** [comment]
<details>
<summary><a href="https://framagenda.org/remote.php/dav/public-calendars/XXXXXXXXXXXXXXXX?export">https://framagenda.org/remote.php/dav/public-calendars/XXXXXXXXXXXXXXXX?export</a>:</summary>

```ics
BEGIN:VCALENDAR
VERSION:2.0
CALSCALE:GREGORIAN
PRODID:-//SabreDAV//SabreDAV//EN
X-WR-CALNAME:Personal (Benjamin Loison)
REFRESH-INTERVAL;VALUE=DURATION:PT4H
X-PUBLISHED-TTL:PT4H
END:VCALENDAR
```
</details>

mentions *SabreDAV*.

**Comment by Benjamin-Loison** [comment]
Maybe can use a redirection to simplify the URL.

**Comment by Benjamin-Loison** [comment]
![Image](https://github.com/user-attachments/assets/19f30149-cda2-4502-bf67-3497b4d41478)

![Image](https://github.com/user-attachments/assets/35c3427b-8212-420c-b9c2-282655a88c5b)

![Image](https://github.com/user-attachments/assets/8e719dbf-4fa1-4cb5-805f-1dc78343d288)

**Comment by Benjamin-Loison** [comment]
![Image](https://github.com/user-attachments/assets/871c267e-893c-4c7c-bb4a-c94988c3f26b)

![Image](https://github.com/user-attachments/assets/81831b53-fb71-4984-911d-24497ac9d664)

![Image](https://github.com/user-attachments/assets/58fafe7d-ce1e-44e0-9b4b-8191d745f09b)

![Image](https://github.com/user-attachments/assets/ee5ab7dd-6a7c-4aa0-b98b-2a3139977bf2)

![Image](https://github.com/user-attachments/assets/97fab8d6-71d4-4658-9c08-8a2644384bfc)

![Image](https://github.com/user-attachments/assets/e7d0d7e5-f216-4e51-b9db-1b103af4f103)

![Image](https://github.com/user-attachments/assets/669ceac8-748b-4d4e-8d62-41f2e757a275)

**Comment by Benjamin-Loison** [comment]
The question is now how to have write access from Etar.

**Comment by Benjamin-Loison** [comment]
https://framagenda.org/apps/calendar/p/XXXXXXXXXXXXXXXX

Source: *Copy public link*

results in above *Configuration detection* issue.

**Comment by Benjamin-Loison** [comment]
![Image](https://github.com/user-attachments/assets/677b9f25-1efb-41c9-b097-2e8f96bc7178)

![Image](https://github.com/user-attachments/assets/b5ca82ce-dbc8-4407-b7d3-2ef22f067114)

![Image](https://github.com/user-attachments/assets/2c6ac710-d253-4981-ad21-7261de4c20a5)

![Image](https://github.com/user-attachments/assets/a1b3144f-eb89-4e67-8912-2aa4d80be5b1)

![Image](https://github.com/user-attachments/assets/89dc4792-48bd-4bb3-8c6d-6dc0ba6aece6)

After DAVx⁵ *Synchronize now* able to write and see the update on each end.

[Benjamin_Loison/davx5-ose/issues/5](https://codeberg.org/Benjamin_Loison/davx5-ose/issues/5) would help, notably to keep the history in any format like a Git on Codeberg.

**Comment by Benjamin-Loison** [comment]
Once move my public events to such calendar, can notify associated persons to let them know that they can now track when I plan to work on it possibly.

**Comment by Benjamin-Loison** [comment]
An online viewer would be nice.

<details>
<summary><i>Copy embedding code</i>:</summary>

```html
<iframe width="400" height="215" src="https://framagenda.org/apps/calendar/embed/XXXXXXXXXXXXXXXX"></iframe>
```
</details>

https://framagenda.org/apps/calendar/embed/XXXXXXXXXXXXXXXX

![Image](https://github.com/user-attachments/assets/eaa46f38-06ec-43c3-8626-630522720f54)

Top right icons allows respectively downloading ICS and giving its link.

Note that concerning the history if have to delete something leaked, there seems to be a *Trash bin*.

**Comment by Benjamin-Loison** [comment]
![Image](https://github.com/user-attachments/assets/63c3567a-b67a-4835-97de-ffc419ac4803)

> [Copy primary CalDAV address](https://framagenda.org/remote.php/dav)
> [Copy iOS/macOS CalDAV address](https://framagenda.org/remote.php/dav/principals/users/Benjamin%20Loison/)

**Comment by Benjamin-Loison** [comment]
Related to [Benjamin_Loison/nextcloud_server/issues/1](https://codeberg.org/Benjamin_Loison/nextcloud_server/issues/1).

**Comment by Benjamin-Loison** [comment]
If move an event from an EteSync calendar to the public one, is the history of EteSync calendar event also moved? This time I do not want it at least as it contains secrets. As far as I know the history aspect is only an EteSync feature, so I doubt that it is passed.

**Comment by Benjamin-Loison** [comment]
Note that https://framagenda.org/apps/calendar/timeGridDay/now is quite updated in real time (no need to refresh webpage) when add or remove an event, maybe needing on DAVx⁵ *Synchronize now*.

**Comment by Benjamin-Loison** [comment]
A nice thing would be to refer to an event with an event specific URL, such that it does not change even if change its date time.

Have to pay attention retrieving event public link from a different account/while not being authenticated to not be redirected to the Framagenda home page:

https://framagenda.org/apps/calendar/timeGridDay/2026-04-01/edit/full/L3JlbW90ZS5waHAvZGF2L2NhbGVuZGFycy9CZW5qYW1pbiUyMExvaXNvbi9wZXJzb25hbC8yOTAxMzBiYi00OTcyLTQwYWItOTIwYi0zYzNiNjg0YzNkMzMuaWNz/1775066400

```
/remote.php/dav/calendars/Benjamin%20Loison/personal/290130bb-4972-40ab-920b-3c3b684c3d33.ics
```

https://framagenda.org/apps/calendar/p/Dg8W8tqd5Dp9KTd3/timeGridDay/2026-04-01/view/full/L3JlbW90ZS5waHAvZGF2L3B1YmxpYy1jYWxlbmRhcnMvRGc4Vzh0cWQ1RHA5S1RkMy8yOTAxMzBiYi00OTcyLTQwYWItOTIwYi0zYzNiNjg0YzNkMzMuaWNz/1775066400

```
/remote.php/dav/public-calendars/Dg8W8tqd5Dp9KTd3/290130bb-4972-40ab-920b-3c3b684c3d33.ics
```

Concernant UUID, see [#issuecomment-2708873224](#issuecomment-2708873224).

Related to [Benjamin_Loison/nextcloud_server/issues/14](https://codeberg.org/Benjamin_Loison/nextcloud_server/issues/14).

**Comment by Benjamin-Loison** [comment]
From my authenticated point of view:

https://framagenda.org/apps/calendar/timeGridDay/now/edit/popover/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/1741528800

the censored part is base 64 encoding:

`/remote.php/dav/calendars/Benjamin%20Loison/personal/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX.ics`

**Comment by Benjamin-Loison** [comment]
*More details* > *Export* leads to https://framagenda.org/remote.php/dav/calendars/Benjamin%20Loison/personal/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX.ics?export

Right-clicking the event does not seem to lead to any specific option.

**Comment by Benjamin-Loison** [comment]
Clicking on event from public view:

https://framagenda.org/apps/calendar/embed/XXXXXXXXXXXXXXXX/dayGridMonth/now/view/sidebar/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/1741528800

the censored part is base64 encoding:

`/remote.php/dav/public-calendars/XXXXXXXXXXXXXXXX/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX.ics`

*...* > *Export*: https://framagenda.org/remote.php/dav/public-calendars/XXXXXXXXXXXXXXXX/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX.ics?export

**Comment by Benjamin-Loison** [comment]
People can easily switch from https://framagenda.org/apps/calendar/embed/XXXXXXXXXXXXXXXX/dayGridMonth/now/view/sidebar/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/1741528800 to my public calendar:

![Image](https://github.com/user-attachments/assets/2ddd0988-6803-4494-bceb-d8d605f0277d)

**Comment by Benjamin-Loison** [comment]
The timestamp at the end of the URL is the event start.

Removing it make the link no more work, so need to update it to still have the link work if change the event start date time, there is no base64 change otherwise.

As `290130bb-4972-40ab-920b-3c3b684c3d33` in [#issuecomment-2708867706](#issuecomment-2708867706) is correctly the `UID`, people can manually find the event even if it has been moved chronologically.

Tracked at [Benjamin_Loison/nextcloud_server/issues/17](https://codeberg.org/Benjamin_Loison/nextcloud_server/issues/17).

**Comment by Benjamin-Loison** [comment]
Thunderbird:

<img width="632" height="786" alt="Image" src="https://github.com/user-attachments/assets/1aade582-88f9-44b6-97ce-978a2b30545a" />

<img width="632" height="786" alt="Image" src="https://github.com/user-attachments/assets/a25fefd4-cfaf-4992-8efc-69459e23c52f" />

It is either none or one of the three proposed entries, as there is an associated tick.

Unclear what these entries precisely mean.

**Comment by Benjamin-Loison** [comment]
https://www.davx5.com/faq/subscribe-ics-file/
https://icsx5.bitfire.at

> read-only calendar

[F-Droid: ICSx⁵ (at.bitfire.icsdroid)](https://f-droid.org/packages/at.bitfire.icsdroid/)

So ICS does not seem to meet my needs.

DuckDuckGo and Google search *Wikipedia list calendar servers*.

[Wikipedia: Category:Calendaring software (730755856)](https://en.wikipedia.org/w/index.php?title=Category:Calendaring_software&oldid=730755856)

does not seem to help.
